app/web/src/components/Navigation/Navigation.tsx

```diff
@@ -259,49 +264,59 @@ export default function Navigation() {
   const menuItems = (
     <div>
       {menuDropDown(data).map(
-        ({ name, title, notificationCount, route, target }) => {
-          if (title)
-            return (
-              <MenuItem classes={{ root: classes.menuTitle }}>{name}</MenuItem>
-            );
-
-          const nameElement = <Typography noWrap>{name}</Typography>;
+        ({
+          name,
+          notificationCount,
+          route,
+          target,
+          externalLink,
+          hasBottomDivider,
+        }) => {
           const hasNotification =
-            notificationCount !== undefined && notificationCount > 0;
+            notificationCount !== undefined && notificationCount > -1;
 
-          return (
-            <MenuItem key={name}>
-              <a
-                href={route}
-                target={target || "_self"}
-                className={`${classes.menuItem} ${
-                  hasNotification && classes.menuItemMessage
-                }`}
-              >
-                {hasNotification ? (
-                  <Badge
-                    color="primary"
-                    variant="dot"
-                    classes={{ badge: classes.badge }}
-                  >
-                    {nameElement}
-                  </Badge>
-                ) : (
-                  nameElement
-                )}
+          const linkContent = (
+            <MenuItem
+              classes={{
+                root: classNames(classes.menuItem, {
+                  [classes.menuItemMessage]: hasNotification,
+                  [classes.menuItemDivider]: hasBottomDivider,
```

Comment:
move these to our `MenuItem` component as I mentioned in my comment

Reply:
Should i create an interface that extends Material UI's "MenuItemProps" type ?

Reply:
![01](https://user-images.githubusercontent.com/62873665/145246908-bdf8a278-c1c9-4f9a-8b8f-0faef4f1edc1.png)

Reply:
I would import the original as `MuiMenuItemProps`, make the interface `MenuItemProps` and extend the existing `Omit` type so you don't need that in the arguments.

Reply:
Okay, ^^ sorry if i don't understood what you meant but is that what you expected ?  :)

![01](https://user-images.githubusercontent.com/62873665/145252608-f726b88e-cec5-4e4c-aadd-a52c939ae617.png)